.github/pull_request_template.md

```diff
@@ -31,8 +31,9 @@ https://raw.githubusercontent.com/ceph/ceph/master/SubmittingPatches.rst
 
 -->
 ## Checklist
-- [ ] References tracker ticket
-- [ ] Updates documentation if necessary
+- [ ] References [tracker ticket](https://tracker.ceph.com/issues/new)
+- [ ] Does not require changes to [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new) and/or [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new) (otherwise, references corresponding tickets)
```

Comment:
Maybe we can phrase is the other way around? Maybe "Changes to the Dashboard are taken care of" or something

Reply:
Agree wording could be in the positive direction - don't want to imply that changes to dashboard/orch should necessarily be part of the same PR though. Maybe 'taken care of or a tracker ticket filed' or something?

Reply:
@sebastian-philipp @jdurgin done! Thanks a lot for the idea.